uber/model_checks.py

```diff
@@ -64,22 +64,40 @@ def _invalid_zip_code(s):
     return len(re.findall(r'\d', s)) not in [5, 9]
 
 
```

Comment:
While fixing this, I broke out most of the validations into their own individual methods.  The reason for this is that we now have a way for event-specific plugins to turn off validation methods, so we want them to be broken down so that it's easy for plugins to disable individual validations instead of accidentally turning off a bunch of unrelated ones.